packages/babel-preset/README.md

```diff
@@ -145,37 +145,25 @@ const useStyles = __styles(/* resolved styles */);
 This section focuses mainly on troubleshooting this babel preset in the [microsoft/fluentui](https://github.com/microsoft/fluentui) repo.
 However, the concepts are not coupled to the repo setup.
 
-### Linaria
+### Module evaluation
 
 The preset uses tools from [linaria](https://github.com/callstack/linaria) to evaluate runtime calls of `makeStyles`.
-[Linaria's debugging documentation can help here](https://github.com/callstack/linaria/blob/master/CONTRIBUTING.md#debugging-and-deep-dive-into-babel-plugin). Here are a few examples with building packages with linaria debug output.
+[Linaria's debugging documentation can help here](https://github.com/callstack/linaria/blob/master/CONTRIBUTING.md#debugging-and-deep-dive-into-babel-plugin).
 
-Directly from the package
+Debugging output can be activated with following environment variables:
 
 ```sh
 $ DEBUG=linaria\* LINARIA_LOG=debug yarn build
 ```
 
-Using Lage from the root of the repo
-
-```sh
-$ DEBUG=linaria\* LINARIA_LOG=debug yarn lage build --to <package-name>
-```
-
-Using `yarn workspace` from the root of the repo
```

Comment:
We don't have `lage` in this repo, third parties may not have it. It's not relevant